File: 0169-majority-element/0169-majority-element.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        unordered_map<int,int> mp;

        for(int i : nums){
            mp[i]++;
        }

        int maxi=0;
        int ans=0;

        for(auto it: mp){
            if(it.second>maxi){
                maxi=it.second;
                ans=it.first;
            }
            
        }

        return ans;
    }
};

File: 0169-majority-element/0169-majority-element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        
        mp={}

        for i in nums:
            mp[i]=mp.get(i,0)+1
        
        maxi=0
        ans=0


        for key,value in mp.items():
            if(value>maxi):
                ans=key
                maxi=value
            
        
        return ans

